kernel/proc.h

```diff
@@ -104,4 +105,6 @@ struct proc {
   struct file *ofile[NOFILE];  // Open files
   struct inode *cwd;           // Current directory
   char name[16];               // Process name (debugging)
+  int tickets;
+  int ticks;
```

Comment:
여기서 uint32로 만들거나, 아니면 여기서 int로 두고 proc.c 에서 관련 변수를 int 로 만들거나, 아무튼 둘중 한쪽으로 타입을 통일해야할것같습니다. 저는 uint32 쪽이 더 나은것같습니다.
```suggestion
  uint32 tickets;
  uint32 ticks;
```